README.md

```diff
@@ -57,10 +57,10 @@ After running `docker-compose up` successfully, the output in your terminal shou
 
     > Note: Comes with a full PostgreSQL installation.
 
-- [GDAL (Geospatial Data Abstraction Library)](GDAL_LIBRARY_PATH) (GIS support for PostgreSQL)
+- [GDAL (Geospatial Data Abstraction Library)] (GIS support for PostgreSQL)
```

Comment:
```suggestion
- GDAL (Geospatial Data Abstraction Library) (GIS support for PostgreSQL)
- ```

Reply:
`[...](...)` is markdown syntax for a hyperlink, removing the brackets since we removed the link as well